test/karma-test-shim.js

```diff
@@ -87,9 +87,38 @@ function configureTestBed() {
 
     jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000;
 
-    testing.TestBed.initTestEnvironment(
+    var testBed = testing.TestBed.initTestEnvironment(
       testingBrowser.BrowserDynamicTestingModule,
       testingBrowser.platformBrowserDynamicTesting()
     );
+
+    applyAngularTestingPatches(testBed);
+  });
+}
+
+/** Applies patches and workarounds to Angular's testing package. */
+function applyAngularTestingPatches(testBed) {
+  // Original resetTestingModule function of the TestBed.
+  var _resetTestingModule = testBed.resetTestingModule;
+
```

Comment:
That's true, but what I mean is that it's very unlikely that the `_activeFixtures` property will be removed or whatever.

If it really comes to an error you will see the file that failed and the according line number. 

Would be `test/karma-test-shim.js:XXX`